### PR TITLE
Markdown output

### LINKED ISSUE #115: Markdown Output
It would be useful if Markdown was added to the available outputs. 
The default table is already great, but if it were possible to have a Markdown table then the results could be rendered nicely without any post processing.

### PR DESCRIPTION
**Pull request by mariusmuntean**
Added basic support to output the results as a markdown table.
Fixes #115

### CODE REVIEW

File: test/endToEnd.test.js [hunk 1]
```diff
@@ -344,3 +360,18 @@ const EXPECTED_TABLE_TEMPLATE = `{{department}}  {{relatedTo}}  {{name}}  {{lice
 {{department}}  {{relatedTo}}  [[lodash]]  {{licensePeriod}}  {{material}}  {{licenseType}}  {{link}}  {{remoteVersion}}  {{installedVersion}}  {{definedVersion}}  {{author}}
 {{department}}  {{relatedTo}}  [[semver]]  {{licensePeriod}}  {{material}}  {{licenseType}}  {{link}}  {{remoteVersion}}  {{installedVersion}}  {{definedVersion}}  {{author}}
 `
+
+/*
+	template for csv output; usage:
+	{{key}} - value to be replaced with value from package information
+	[[package-name]] - name of the package
+*/
+const EXPECTED_MARKDOWN_TABLE_TEMPLATE =
```
Comment (BePo65) [inline test/endToEnd.test.js:369]: I am missing the placeholders in the 'EXPECTED_MARKDOWN_TABLE_TEMPLATE' (comparable to the 'EXPECTED_TABLE_TEMPLATE'). This way, any test using this constant as an expected value will fail, as soon as a newer version is published in the npm repository.

**Comment by BePo65** [comment]
Sorry for the delay, but it's been only today that I finished a quite complex pr for the 'dayjs' package.

Regarding the configuration, I think we could simply add an entry to the `config.js` e.g. named `tablemarkOptions` with an empty literal object (`{}`) as a default value. If not empty, we could add this as a second parameter to the line 109 in `lib/getFormatter.js` ('return tablemark(dataAsArrayWithRenamedFields)').  
By default, does not require any configuration, as it uses the field names as column titles.

What do you think about this?

**Comment by mariusmuntean** [comment]
I fully agree and I'll update the pr tomorrow or so.

**Comment by mariusmuntean** [comment]
@BePo65 I've added the tablemark options passthrough. It wasn't necessary to check if the tablemark options object is empty.
I've also updated the README.md a bit.
What do you think?

**Review by BePo65 — COMMENTED** [review]
Sorry, but I forgot to **submit** my review (silly error 😄 ).
Would you please add the placeholders in the template of the test results?

The rest is very well (even with good comments 👍 ).

**Comment by BePo65** [comment]
Thanks for adding the configuration section and especially of thinking of the README.md file.

File: test/endToEnd.test.js (hunk 1 above)
Comment (mariusmuntean) [reply test/endToEnd.test.js:369]: I missed that, but it should be fixed now.

**Comment by mariusmuntean** [comment]
I saw the placeholders in the templates initially, but then completely forgot about them.

**Review by BePo65 — APPROVED** [review]
looks good to me

**Pull request merged**